### PR TITLE
Add stop script

### PR DESCRIPTION
**Pull request by louisguitton**
Hi,
image

### CODE REVIEW

**Review by 10thfloor — APPROVED** [review]
Necessary!

File: package.json [hunk 1]
```diff
@@ -30,7 +30,8 @@
     "update:emulator": "flow project deploy --update",
     "update:testnet": "npx dotenv-cli -e .env.testnet -- flow project deploy --network=testnet -f flow.json -f flow.testnet.json --update",
     "dev:reset": "cross-env node .ki-scripts/reset.js",
-    "postinstall": "npx lerna exec npm install"
+    "postinstall": "npx lerna exec npm install",
+    "stop": "npx pm2 stop dev-wallet emulator api web"
```
Comment (10thfloor) [inline package.json:34]:
```suggestion
    "stop": "npx pm2 stop all"
```

Might be better to stop all just to be safe?

**Review by 10thfloor — CHANGES_REQUESTED** [review]
I think you can just do `pm2 stop all`
Might be b

**Review by 10thfloor — APPROVED** [review]
Too easy.

**Comment by MrDSGC** [comment]
Thank you for improving the codebase @louisguitton! :)

**Pull request merged**